tsconfig.json

```diff
@@ -15,6 +15,12 @@
     "allowSyntheticDefaultImports": true,
     "types": ["webpack-env", "app/javascript/declarations"]
   },
-  "exclude": ["**/*.spec.ts", "node_modules", "vendor", "public"],
+  "exclude": [
+    "**/*.spec.ts",
```

Comment:
If you want the _exact_ same, you probably want `test/javascript/**/*.spec.ts`, but I am cool with this.